invenio_sipstore/models.py

```diff
@@ -155,6 +175,31 @@ def validate_key(self, filepath, filepath_):
     """Relation to the SIP along which given file was submitted."""
 
 
+class SIPMetadata(db.Model, Timestamp):
+    """Extra SIP info regarding metadata."""
+
+    __tablename__ = 'sipstore_sipmetadata'
+
+    id = db.Column(UUIDType, primary_key=True, default=uuid.uuid4)
+    """Id of metadata."""
+
+    sip_id = db.Column(UUIDType,
+                       db.ForeignKey(SIP.id, name='fk_sipmetadata_sip_id'))
+    """Id of SIP."""
+
+    format = db.Column(db.String(7), nullable=False)
```

Comment:
I took it from here: https://github.com/inveniosoftware/invenio-sipstore/pull/27/files/34dd3ee95befda08384a34bc26818fb05fa66937#diff-a6c01358c922643bed721b952a78b8c6L59

you should just store 'json' or 'marcxml'...